Tests/mocks/Wraps.h

```diff
@@ -10,6 +10,7 @@ class WrapsImpl {
     virtual FILE* popen(const char* command, const char* type) = 0;
     virtual int pclose(FILE *pipe) = 0;
     virtual void syslog(int pri, const char* fmt, va_list args) = 0;
+	virtual FILE* fopen(const char* filename, const char* mode) = 0;
```

Comment:
Pls fix the indent in many places. Looks like it's taken 4 more spaces or a extra tab. There are other places in the PR that needs to be fixed

Reply:
Thanks for the feedback.
The alignment issues in all the files have been resolved now.